src/cosmos/Swap.fixture.tsx

```diff
@@ -65,44 +98,66 @@ function Fixture() {
     'Mainnet only': tokens.filter((token) => token.chainId === SupportedChainId.MAINNET),
   }
   const tokenList = useOption('tokenList', { options: tokenLists, defaultValue: 'Default', nullable: false })
-  console.log(tokenList)
 
   const [routerUrl] = useValue('routerUrl', { defaultValue: 'https://api.uniswap.org/v1/' })
 
   return (
-    <SwapWidget
-      convenienceFee={convenienceFee}
-      convenienceFeeRecipient={convenienceFeeRecipient}
-      defaultInputTokenAddress={defaultInputToken}
-      defaultInputAmount={defaultInputAmount}
-      defaultOutputTokenAddress={defaultOutputToken}
-      defaultOutputAmount={defaultOutputAmount}
-      hideConnectionUI={hideConnectionUI}
-      locale={locale}
-      jsonRpcUrlMap={jsonRpcUrlMap}
-      defaultChainId={defaultChainId}
-      provider={connector}
-      theme={theme}
-      tokenList={tokenList}
-      width={width}
-      routerUrl={routerUrl}
-      onConnectWalletClick={() =>
-        new Promise((resolve) => {
-          console.log('integrator provided a onConnectWalletClick')
-          resolve(true) // to open our built-in wallet connect flow
-        })
-      }
-      onReviewSwapClick={() => new Promise((resolve) => resolve(true))}
-      onTokenSelectorClick={(f: Field) =>
-        new Promise((resolve) => {
-          console.log('onTokenSelectorClick', f)
-          resolve(true)
-        })
-      }
-      onTxSubmit={(txHash: string, data: any) => console.log('tx submitted:', txHash, data)}
-      onTxSuccess={(txHash: string, data: any) => console.log('tx success:', txHash, data)}
-      onTxFail={(error: Error, data: any) => console.log('tx fail:', error, data)}
-    />
+    <Row align="baseline" justify="space-around">
+      <SwapWidget
+        convenienceFee={convenienceFee}
+        convenienceFeeRecipient={convenienceFeeRecipient}
+        defaultInputTokenAddress={defaultInputToken}
+        defaultInputAmount={defaultInputAmount}
+        defaultOutputTokenAddress={defaultOutputToken}
+        defaultOutputAmount={defaultOutputAmount}
+        hideConnectionUI={hideConnectionUI}
+        locale={locale}
+        jsonRpcUrlMap={jsonRpcUrlMap}
+        defaultChainId={defaultChainId}
+        provider={connector}
+        theme={theme}
+        tokenList={tokenList}
+        width={width}
+        routerUrl={routerUrl}
+        onConnectWalletClick={() =>
```

Comment:
Simplify the Promise-returning event handlers as

    onConnectWalletClick={() => {
      addEvent({ message: 'onConnectWalletClick', data: {} })
      return Promise.resolve(true)
    }}